### DISCUSSION TITLE
[@types/zingchart] graphsets.plot.selected-state.line-color property not defined

### DISCUSSION THREAD

**Discussion by danielpza** [DefinitelyTyped/DefinitelyTyped#55561] [Issues with a @types package]
The `graphsets.plot.selected-state.line-color` and possibly others are not defined. I'm getting an error when trying to use it. See https://www.zingchart.com/docs/api/json-configuration/graphset/plot/selected-state

[a comment by an automated account is not shown]

**Comment by danielpza** [comment]
Other properties not included detected by typescript:

- graphset.zoom.shared
- graphset.crosshair-x (actual type is graphset.crosshair)
- graphset.crosshair-y (actual type is graphset.crosshair)

**Comment by danielpza** [comment]
related https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52892

**Comment by jeanettephung** [comment]
The following type definitions have been added:
- graphsets.plot.selected-state.line-color
- graphset.zoom.shared
- graphset.crosshair-x
- graphset.crosshair-y
